src/main/java/org/isf/lab/service/LabIoOperationRepository.java

```diff
@@ -26,9 +26,10 @@
 
 import org.isf.lab.model.Laboratory;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
```

Comment:
These two imports are not used.  @ArnaudFonzam

Reply:
Done, thanks!